pkg/controller/replication/replication_controller.go

```diff
@@ -575,9 +576,6 @@ func (rm *ReplicationManager) syncReplicationController(key string) error {
 	cm := controller.NewPodControllerRefManager(rm.podControl, rc, labels.Set(rc.Spec.Selector).AsSelectorPreValidated(), controllerKind)
 	filteredPods, err = cm.ClaimPods(pods)
 	if err != nil {
-		// Something went wrong with adoption or release.
```

Comment:
There is one additional enqueue in the error check above this one that is not needed.